.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+
+python:
+    - "3.6"
+
+services: postgresql
+
+env:
+    - DJANGO=2.0.6 DB=postgresql
+    
+install:
+    - pip install -r requirements.txt
+
+before_script:
+    - psql -c "CREATE DATABASE test;" -U postgres
+
+script:
+    - python manage.py test
+    
```

Comment:
End of line

Reply:
I don't exactly understand what you mean by *End of line*, kindly elaborate more on that.